Scripts/Python/ki/__init__.py

```diff
@@ -1655,13 +1655,18 @@ def DetermineTextColor(self):
         if entry is not None:
             colorStr = entry.chronicleGetValue()
             PtDebugPrint(f"xKI.DetermineTextColor(): KI Text Color is: \"{colorStr}\".", level=kWarningLevel)
-            args = colorStr.split(",")
-            try:
-                self.chatMgr.chatTextColor = ptColor(float(args[0]), float(args[1]), float(args[2]))
-            except (IndexError, ValueError):
-                # format is incorrect -- didn't have 3 values, or values weren't floats -- so log an error and destroy all evidence
-                PtDebugPrint(f"xKI.DetermineTextColor(): KI Text Color was not in the expected triplet format", level=kWarningLevel)
-                vault.addChronicleEntry(kChronicleKITextColor, kChronicleKITextColorType, "")
+            if colorStr:
```

Comment:
Unfortunately, I can't make suggestions across deleted lines, but it might make more sense to do:
```python
if colorStr := entry.chronicleGetValue():
```
so you know that `colorStr` is non-empty, and it's all once nice line.